packages/react-sdk/src/components/Layout/useFullscreenMode.ts

```diff
@@ -55,10 +55,8 @@ export function useFullscreenMode(): UseFullscreenModeResult {
     isFullscreenRequestRunning.current = true;
 
     if (value && !document.fullscreenElement) {
-      const container =
-        document.getElementById('widget-root') ?? document.documentElement;
       try {
-        await container.requestFullscreen();
+        await document.body.requestFullscreen();
```

Comment:
This change breaks fullscreen in NeoBoard standalone:

![image](https://github.com/user-attachments/assets/da7fabc5-aac2-4a10-bd90-d389db54afd3)

It should only show the canvas full screen.